README.md

```diff
@@ -122,6 +122,12 @@ other sections.
 - `otp_device` - Allow the automatic selection of an OTP device.  
   This value is the human readable string name for the device.
   Eg, `OneLogin Protect`, `Yubico YubiKey`, etc
+- `ip_address` - The client IP address to send to OneLogin.
+  Relevant when using OneLogin Policies with an IP whitelist.
+  If this is specified, `auto_determine_ip_address` is not used.
```

Comment:
If the commands are mutually exclusive, we can stop them being provide and give some error handling with a mutually exclusive arg group. There's an example from the `renew-seconds` options:

https://github.com/physera/onelogin-aws-cli/pull/113/files#diff-f857b4b87f1d478f36a272425e1e614aL51

Reply:
That method seems like it only applies to CLI params, not config options?

Reply:
That is true, yes. It's an easy win for some minimum error checking. Just a thought. :-)